src/c_api/c_api.cc

```diff
@@ -1363,7 +1363,15 @@ int MXGetVersion(int *out) {
 #if MXNET_USE_TVM_OP
 int MXLoadTVMOp(const char *libpath) {
   API_BEGIN();
-  tvm::runtime::TVMOpModule::Get()->Load(libpath);
+  tvm::runtime::TVMOpModule *libpath_module =  tvm::runtime::TVMOpModule::Get();
+  libpath_module->Load(libpath);
+#if MXNET_USE_CUDA
+  std::string libpathstr(libpath);
+  std::string cubinpath = libpathstr.substr(0, libpathstr.size() - 11) + "libtvmop.cubin";
```

Comment:
would be better to pass `libpath` as dir, and do libpath + "libtvmop.so" as well to keep consistency.

Reply:
Yes, but `MXLoadTVMOp` is called at:
https://github.com/apache/incubator-mxnet/blob/1bf881f381f91b157a26d9beddcaa8f4960cc038/python/mxnet/tvmop.py#L31-L32
where `_LIB_TVM_OP` is returned from the 3rdparty TVM function [`find_lib_path`](https://github.com/apache/incubator-tvm/blob/ae119f81e714e302a813d2440d288efa4c8a35e3/python/tvm/_ffi/libinfo.py#L42-L142), and `_LIB_TVM_OP[0]` is the path of `libtvmop.so`.
We may need to modify `find_lib_path` or write a new function to get the directory that `libtvmop.so` locates.